pkgs/desktops/gnome-3/core/mutter/3.34/default.nix

```diff
@@ -105,47 +104,14 @@ stdenv.mkDerivation rec {
   ];
 
   patches = [
-    # Fix build with libglvnd provided headers
-    (fetchpatch {
-      url = "https://gitlab.gnome.org/GNOME/mutter/commit/a444a4c5f58ea516ad3cd9d6ddc0056c3ca9bc90.patch";
-      sha256 = "0imy2j8af9477jliwdq4jc40yw1cifsjjf196gnmwxr9rkj0hbrd";
-    })
-
-    # gnome-3-34 2020-04-24
-    # also fixes https://mail.gnome.org/archives/distributor-list/2020-April/msg00001.html
-    (fetchpatch {
-      url = "https://github.com/GNOME/mutter/compare/3.34.5..3bafd234248fdcd84bc62fef5e31c29fbb613909.patch";
-      sha256 = "1a7krbdfmvx204p6av44rbp4ckp6ddg1mms8wkixxh2p871zq1pi";
-    })
-
-    # Drop inheritable cap_sys_nice, to prevent the ambient set from leaking
-    # from mutter/gnome-shell, see https://github.com/NixOS/nixpkgs/issues/71381
-    ./drop-inheritable.patch
-
-    # See commit message for details
-    ./0001-Fix-glitches-in-gala.patch
-
-    # https://gitlab.gnome.org/GNOME/mutter/merge_requests/1094
-    # https://gitlab.gnome.org/GNOME/mutter/merge_requests/957
-    ./fix-sysprof.patch
-
-    # profiler: track changes in GLib and Sysprof
-    # https://gitlab.gnome.org/GNOME/mutter/merge_requests/908
-    (fetchpatch {
-      url = "https://gitlab.gnome.org/GNOME/mutter/-/commit/605171291993460f31d470a8143d6438d0c6169c.patch";
-      sha256 = "10fxzj0lmic2sp57w26w3r0bv1szngjjs50p3ka22wr9pxqmzl7l";
-    })
-
-    # Fixes https://github.com/elementary/wingpanel/issues/305
-    (fetchpatch {
-      url = "https://gitlab.gnome.org/GNOME/mutter/-/commit/9d390ee49fb1f6300336e82ae94cc8061c6bae12.patch";
-      sha256 = "12hmi07rvspwhp8h1y1vmcvmvbh8fihcrb07ja5g0qnh28ip5qfi";
-    })
-
-    (substituteAll {
-      src = ./fix-paths.patch;
-      inherit zenity;
-    })
+    ./0001-EGL-Include-EGL-eglmesaext.h.patch
+    ./0002-drop-inheritable.patch
+    ./0003-Fix-glitches-in-gala.patch
+    ./0004-build-use-get_pkgconfig_variable-for-sysprof-dbusdir.patch
+    ./0005-profiler-track-changes-in-GLib-and-Sysprof.patch
+    ./0006-meta-Add-missing-display.h-to-meta-workspace-manager.h.patch
+    ./0007-build-bump-ABI-to-sysprof-capture-4.patch
+    ./0008-fix-paths.patch
```

Comment:
This seems to be missing `substituteAll`.

Reply:
Oh yeah, forgot about that. Will fix.